### ISSUE TITLE
Interventions

### ISSUE THREAD

**Issue by alexgoodell** [alexgoodell/go-mdism#53]
Could you explain your thoughts on how to implement the fructose interventions

**Comment by Vreman** [comment]
Yes. So I think it will be complicated, and it depends a lot on the order of how we implement.

So the way it works now, is that for each intervention the percentage of people that consume sugar above the cut-off value is different.
I have put this in the ras file, but this did not change the original fact that, for each intervention, the input value for the probability from uninitialized -> increased risk, will be different. In the ras file we just have to adjust more values (instead of just 1) to derive the new probabilities.

The problem that is inherent to this fact, is that we cannot concurrently run these interventions, because their inputs are different. We will have to run them in sequence (as you clarified).

We want to run the PSAfunction to derive new input parameters for basically every input, and then when we run an intervention, we want these new values also adjusted for the percentage-change of being at high risk due to excessive sugar intake. 

We will have to store all the data that we have found after running the PSAfunction (that goes for all of the input files!) to use them when we run the interventions. We change some of these values during a simulation (TPs for instance), so we cannot just use what we have at the end of the simulation, we need to use the starting values from before we started the simulation without intervention.

Then for the values of sugar consumption with each intervention, I will make an input file that says by what amount (percentage) we need to adjust the probability of being high risk. We can adjust the values that we got after the PSAfunction by this percentage and then paste those into our ras file before we start simulating. Then when we do intervention2, we can adjust them again (the PSAvalues at beginning are still saved), but by a different percentage, to run the second intervention.

So all we need is a way to save all the input data after we ran the PSA function on all of them, and the file for intervention, that specifies the precentages to adjust each ras-probability for increased risk by.

Agreed? Or will this not work?
